### DISCUSSION TITLE
Post Processing Effects -  SSR

### DISCUSSION THREAD

**Discussion by WadeMorrison** [google/model-viewer#4321] [Q&A] [answered]
I noticed that the [documentation](https://modelviewer.dev/examples/postprocessing/) doesn't list any support for SSR out of the box, so would this just need to be done with the addPass() method? Just wanting to make sure I understand it correctly, or maybe there is support for it and I happened to have missed it.

**Comment by diegoteran** [comment, marked as answer]
You are right, we currently don't support Screen Space Reflection and you'll need to use addPass() in https://modelviewer.dev/examples/postprocessing/#custom-effects

**Reply by WadeMorrison** [reply]
Perfect, thank you!

**Reply by Winny80** [reply]
Dear both,
do you mind sharing how you made it work?
We want to add SSR to our project as well.
Many thanks
